x-pack/plugins/security_solution/public/flyout/document_details/alert_preview_button.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonIcon } from '@elastic/eui';
+import { useExpandableFlyoutApi } from '@kbn/expandable-flyout';
+import React, { useCallback } from 'react';
+import type { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { ALERT_PREVIEW_BANNER } from './preview/constants';
+import { DocumentDetailsPreviewPanelKey } from './shared/constants/panel_keys';
+
+interface AlertPreviewButtonProps {
+  /**
+   * Name of the index used in the parent's page
+   */
+  indexName: string;
+  id: string;
+  'data-test-subj'?: string;
+  scopeId: string;
```

Comment:
can you add a short description for each property here, just to stay consistent with the rest of the components under the `flyout/document_details` folder. You can find a description for each of the fields in nearby components